decidim-elections/app/commands/decidim/elections/admin/setup_election.rb

```diff
@@ -135,6 +135,13 @@ def notify_trustee_about_election
 
           Decidim::EventsManager.publish(data)
         end
+
+        def flatten_translations(translated_attribute)
```

Comment:
I don't fully understand what is this doing, could you explain it?

Reply:
The `machine_translations` used to return a nested hash:
```
description: {
  "ca"=>"Repudiandae dolorem.",
  "en"=>"Doloribus rerum.",
  "machine_translations"=> {
    "es"=>"Nobis repellat."
   }
}
```
This will result in an error on the `bulletin_board` side as `Electionguard` and other schemes that follow the [Civics Common Standard Data Specification](https://developers.google.com/civics-data/reference/internationalized-text) expect the translations to be returned in a "simple" hash. For that, we removed the `machine_translations` field and only returned the value of that nested hash:
```
  {
    :es=>"Nobis repellat.", 
    :ca=>"Repudiandae dolorem.", 
    :en=>"Doloribus rerum."
  }
```

Reply:
Could you please add documentation about this then?